README.md

```diff
@@ -1 +1,124 @@
-Test
\ No newline at end of file
+# Trustly Parliament Challenge
+
+## Problem
+
+This is a challenge by Trustly known as the "Parliament Challenge".
+
+The Swedish parliament has an open API platform for developers to use its resources to create applications. Among its resources are the speeches in the parliament and the data of its members.
+
+The challenge by Trustly is to create an appliction that serves requests for parliament speeches. The expected response should be created by merging the data from the speeches api and members api and should be in a JSON format.
+
+The expected response should at least contain:
+
+- A unique ID, representing the speech itself (anforande_id)
+- Date of speech (dok_datum)
+- The name of the speaker, and only the name (tilltalsnamn)
+- Political affiliation (parti)
+- The official e-mail address (uppgift)
+- Constituency (valkrets)
+- A decent sized image that could be used in a web site listing (bild_url_192)
+- The debate subject (avsnittsrubrik)
+- A link to the speech (HATEOAS style) (protokoll_url_www)
+
+The link to the challenge can be found on https://github.com/trustly/parliament-challenge
+
+## Approach
+
+The challenge is approached as follows:
+
+1. The function `get_speeches()` is responsible for getting the speeches resources from its respective apis. The speeches api has a filter to extract the ten latest speeches. Since the data is in a nested dict, the for loop has been used to iterate over to the required dict which is "anforande". Speech api: http://data.riksdagen.se/anforandelista/?anftyp=Nej&sz=10&utformat=json
+
+2. The function `get_members()` is responsible for getting the members resources from its respective apis. Here all members data has been extracted. Since the data is in a nested dict, the for loop has been used to iterate over to the required dict which is "person". Members api: http://data.riksdagen.se/personlista/?iid=&fnamn=&enamn=&f_ar=&kn=&parti=&valkrets=&rdlstatus=&org=&utformat=json&sort=sorteringsnamn&sortorder=asc&termlista=
+
+3. Now since the ten latest speeches are in the `get_speeches()` function, a list of dict has been created in the `create_speeches_dict()` function contianing only the relevant keys and values of speeches. An additional key and value `intressent_id` has been added which can be used as a foreign key and be useful for merging the speeches data to members data. The relevant keys are:
```

Comment:
`foreign key` is only a terminology used in data bases. You can instead use the word "reference".

Reply:
used reference id instead